incydr/cli/cmds/audit_log.py

```diff
@@ -0,0 +1,180 @@
+import os
+from typing import Optional
+
+import click
+from click import Context
+
+from incydr._audit_log.models import DateRange
+from incydr._audit_log.models import QueryAuditLogRequest
+from incydr._queries.utils import parse_timestamp_to_posix_timestamp
+from incydr.cli import init_client
+from incydr.cli import log_file_option
+from incydr.cli import log_level_option
+from incydr.cli.cmds.options.audit_log_filter_options import filter_options
+from incydr.cli.cmds.options.output_options import columns_option
+from incydr.cli.cmds.options.output_options import output_options
+from incydr.cli.cmds.options.output_options import table_format_option
+from incydr.cli.cmds.options.output_options import TableFormat
+from incydr.cli.cmds.utils import output_format_logger
+from incydr.cli.cmds.utils import output_response_format
+from incydr.cli.core import IncydrCommand
+from incydr.cli.core import IncydrGroup
+
+
+@click.group(cls=IncydrGroup)
+@log_level_option
+@log_file_option
+@click.pass_context
+def audit_log(ctx, log_level, log_file):
+    """View audit log events."""
+    init_client(ctx, log_level, log_file)
+
+
+# TODO - do we need 'result count' in the CLI?
```

Comment:
I omitted the `get results count` API because I don't think there's a real use case for that.  LMK if you disagree though